helper/runner.go

```diff
@@ -187,9 +188,16 @@ func (r *Runner) EvaluateExpr(expr hcl.Expression, ret interface{}, wantTy *cty.
 	for _, variable := range r.tfconfig.Module.Variables {
 		variables[variable.Name] = variable.Default
 	}
+	workspace, success := os.LookupEnv("TERRAFORM_WORKSPACE")
```

Comment:
According to the official behavior, `TF_WORKSPACE` may be better.
https://www.terraform.io/docs/cli/config/environment-variables.html#tf_workspace